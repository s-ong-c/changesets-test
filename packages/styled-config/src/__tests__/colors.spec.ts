import { colors, primaryColors } from '../index'

describe('primary colors', () => {
  it('light', () => {
    expect(
      primaryColors(colors.light.scheme, colors.light.app)
    ).toMatchSnapshot()
  })
})
