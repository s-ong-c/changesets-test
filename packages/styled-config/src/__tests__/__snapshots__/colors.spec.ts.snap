// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`primary colors light 1`] = `
Object {
  "backgroundLow": "#F5F6F8",
  "blue5": "#F1F8FF",
  "white": "#ffffff",
}
`;
